schematics/ng-add/index.ts

```diff
@@ -152,10 +152,10 @@ function registerLocaleData(moduleSource: ts.SourceFile, modulePath: string, loc
     : new ReplaceChange(modulePath, registerLocaleDataFun[ 0 ].getStart(), registerLocaleDataFun[ 0 ].getText(), `registerLocaleData(${locale});`);
 }
 
-/** 降级 less */
-function downgradeLess(): (host: Tree) => Tree {
```

Comment:
It is more of a Dev UX decision.

If we promote that the one-liner
```
ng add ng-zorro-antd
```
is ready for user to develop on antd and customize their own theme, then we should help them install `less` if they have not installed it before.

If not, we should specify on the docs that
> If you would like to customize theme, install `less` and import the `antd.less`

Personally I am lean to the second approach because you only pay for what you want.